website/project/views/register.py

```diff
@@ -1,19 +1,18 @@
 # -*- coding: utf-8 -*-
 import json
 import httplib as http
-from dateutil.parser import parse as parse_date
 import itertools
 
 from flask import request
 from modularodm import Q
-from modularodm.exceptions import NoResultsFound, ValidationValueError
+from modularodm.exceptions import NoResultsFound
 
+from framework import status
 from framework.exceptions import HTTPError
 from framework.flask import redirect  # VOL-aware redirect
 
-from framework.status import push_status_message
 from framework.mongo.utils import to_mongo
-from framework.forms.utils import process_payload, unprocess_payload
+from framework.forms.utils import unprocess_payload
```

Comment:
Why is `unprocess_payload` used in this module? (it appears to be for url-encoded data)